packages/appium/docs/en/intro/history.md

```diff
@@ -177,6 +177,6 @@ is currently an Impact Project in the foundation.
 
 ## Appium 2.0
 
-Appium 2.0 was released in 2022, with a new focus on Appium as an ecosystem rather than a singular
+Appium 2 was released in 2023, with a new focus on Appium as an ecosystem rather than a singular
```

Comment:
why the year was changed?

Reply:
Maybe I'm missing something, but I have no idea why this was set to 2022. The first beta came out in 2021, and the full 2.0 was released this July.

Reply:
yeah, you're right.